cdalzell/Lahman
# Pkgdown initial site

Created the initial `pkgdown` site (in `docs/`). Looks pretty good for a first stab with minimal configuration.
To test this:

* Merge `pkgdown` branch into `master` via this PR
* @cdalzell : 
   - Go to Settings on the right side of the top menu line
   - Select `Pages` 
   -    -> Source: should be `Deploy from branch`
   -    -> Build & deploy -> Branch: Should be `master`, and `docs/`
* the site should deploy to `https://cdalzell.github.io/Lahman`
* Change the CRAN link to this in the repo
